packages/editor-ui/src/constants.ts

```diff
@@ -139,3 +139,24 @@ export const OTHER_COMPANY_INDUSTRY_KEY = 'otherCompanyIndustry';
 export const VALID_EMAIL_REGEX = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
 export const LOCAL_STORAGE_ACTIVATION_FLAG = 'N8N_HIDE_ACTIVATION_ALERT';
 
+export const HIRING_BANNER = `
+                                                                    //////
+                                                                 ///////////
+                                                               /////      ////
+                                               ///////////////////         ////
+                                             //////////////////////       ////
+     ///////               ///////          ////                /////////////
+  ////////////          ////////////       ////                    ///////
+ ////       ////       ////       ////    ////
+/////        /////////////         //////////
+ /////     ////       ////       ////     ////
+  ////////////          ////////////       ////           ////////
+    ///////                //////           ////        /////////////
+                                             /////////////        ////
+                                                //////////        ////
+                                                       ////      ////
+                                                        ///////////
+                                                          //////
+
+Love n8n? Help us build the future of automation! https://apply.workable.com/n8n/
```

Comment:
can you use https://n8n.io/careers instead? this redirects to workable so that we don't have to update this reference in the future